### DISCUSSION TITLE
Shadows in Hololens 2

### DISCUSSION THREAD

**Discussion by nmitrak** [microsoft/MixedRealityToolkit-Unity#11763] [Q&A]
## Describe the bug
Hi all. I am a newbie at designing apps in hololens 2 so i'll give it a try
Using a point light source renders shadows between objects in Unity but shadows do not render 

## Expected behavior
I expected shadow casting from one object to another

## Screenshots
![εικόνα](https://github.com/microsoft/MixedRealityToolkit-Unity/assets/141948148/0aea37ef-2095-4205-9e5a-c91154805136)
![εικόνα](https://github.com/microsoft/MixedRealityToolkit-Unity/assets/141948148/e16ae75e-375c-4f41-be07-3adbc7156ad1)

## Your setup (please complete the following information)

- Unity Version 2021.3.18f1
- MRTK Version Foundation 2.8.3

## Target platform (please complete the following information)

- HoloLens 2

Thank you

**Comment by AMollis** [comment]
@nmitrak, thank you for your issue.  Unfortunately, this is not directly related to MRTK.   However, I will convert this to a topic in our discuss forums.  Perhaps someone there can assist.

**Comment by nmitrak** [comment]
Ok just a little update. I used the Universal Renderer Pipeline (URP) and upgraded MRTK standard shader for usage with URP. All shadows are now rendered when using hololens 2.
Thank you
